server/app/test/v2/users/controller.spec.js

```diff
@@ -0,0 +1,93 @@
+import { expect } from 'chai';
+import request from 'supertest-as-promised';
+import app from '../../../index';
+import '../../util';
+
+describe('User routes', function userRoutes() {
+  describe('#signup', function signUp() {
+    const signup = '/api/v1/users';
+    const user = {
+      email: 'test@test.com',
+      password: 'password',
+      firstName: 'John',
+      lastName: 'Doe'
+    };
+
+    it('should send a token with valid user', function createUser() {
+      return request(app)
+        .post(signup)
+        .send(user)
+        .expect(200)
+        .then(function onResponse(res) {
+          expect(res.body).to.have.property('token');
+        })
+        .catch(function onError(err) {
```

Comment:
Is there a way to put the test files into a `distServerTest` folder? This way we can still utilize ES6 on the tests as well.

Or did you do this instead to help with debugging by giving the anonymous functions a name?